sdk/go/core/ydk/path/path.go

```diff
@@ -406,17 +508,17 @@ func getDataNodeFromEntity(state *types.State, entity types.Entity, rootSchema C
 func walkChildren(state *types.State, entity types.Entity, dataNode C.DataNode) {
 	children := entity.GetChildren()
 
-	fmt.Printf("Got %d entity children\n", len(children))
+	ydk.YLogInfo(fmt.Sprintf("Got %d entity children", len(children)))
```

Comment:
These should be `YLogDebug`  instead of `YLogInfo`. `Info` should only be used for end-user relevant logs. These messages are more relevant for ydk developers, not end user.